frontend/src/components/Norm/NormMetadataFields.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import MetadataField from "~/components/MetadataField.vue";
+import { isPrototypeProfile } from "~/utils/config";
+
+withDefaults(
```

Comment:
Since we are using Vue 3.5, we could also use JavaScript's native default value syntax, see https://vuejs.org/api/sfc-script-setup.html#reactive-props-destructure